src/web/app/pages-instructor/instructor-session-individual-extension-page/individual-extension-confirm-modal/individual-extension-confirm-modal.component.html

```diff
@@ -0,0 +1,161 @@
+<!-- <div class="modal-header alert-danger"> -->
+
+<div class="modal-header" [ngClass]="{'alert-primary': isExtendModal(), 'alert-danger' : isDeleteModal()}">
+  <h5 class="modal-title" *ngIf="isExtendModal()">Confirm extension of feedback session</h5>
+  <h5 class="modal-title" *ngIf="isDeleteModal()">Confirm deletion of feedback session extensions</h5>
+  <button type="button" class="close" (click)="activeModal.dismiss()">
+    <i class="fas fa-times"></i>
+  </button>
+</div>
+
+<div class="modal-body" >
+  <div *ngIf="isExtendModal()">
+    The new deadline of <b>{{ extensionTimestamp | formatDateDetail: feedbackSessionTimeZone }}</b> will be applied
+    to the following <b>{{selectedStudents.length}}</b> student(s) and <b>{{selectedInstructors.length}}</b> instructor(s) upon confirmation:
```

Comment:
I don't think we need the words "on confirmation"? I don't think this adds much value to the sentence.